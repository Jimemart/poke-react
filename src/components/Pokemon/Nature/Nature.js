import React from "react";
import { Pill } from "../../../UI";

export const Nature = props => {
  const { name, background } = props;

  return <Pill data-hook="nature" background={background}>{name}</Pill>;
};
