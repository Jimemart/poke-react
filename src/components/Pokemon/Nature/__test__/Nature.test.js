import React from "react";
import renderer from "react-test-renderer";
import { Nature } from "../Nature";

describe("Nature Component", () => {
  const props = {
    nature: "grass",
    background: "green"
  };

  it("should render the natures", () => {
    const tree = renderer.create(
      <Nature {...props}>
        <div>Some nested content</div>
      </Nature>
    );

    expect(tree).toMatchSnapshot();
  });
});
