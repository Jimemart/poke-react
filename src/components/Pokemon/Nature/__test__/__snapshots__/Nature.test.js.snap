// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nature Component should render the natures 1`] = `
<div
  className="sc-ifAKCX kwKGvC"
/>
`;
